hwdb.d/70-mouse.hwdb

```diff
@@ -288,6 +288,10 @@ mouse:usb:v04b3p3107:name:*
 mouse:usb:v047dp1020:*Kensington Expert Mouse*
  ID_INPUT_TRACKBALL=1
 
+# Kensington SlimBlade Trackball
+mouse:usb:v047dp2041:name:Kensington Slimblade Trackball:
+ ID_INPUT_TRACKBALL=1
+
```

Comment:
We already have a generic entry in line 166 for anything with `Trackball` in it:
```
mouse:*:name:*Trackball*:
```
how comes this didn't work for you? We shouldn't need this entry, the previous Kensington entry is just there because it *doesn't* have Trackball in its name.

Reply:
Duh. Hmm, so `systemd-hwdb query 'mouse:usb:v047dp2041:name:Kensington Slimblade Trackball:'` **without** this entry already returns `ID_INPUT_TRACKBALL=1`. It seems the matching is happening as it should. If that is confirmed, we should revert this to avoid noise in the db.

Reply:
Yes, you can revert it then. I used it to try to set the scrolling speed and thought the entry could help others if they wanted to make other local modifications only to that trackball.

Reply:
OK, I reverted it now.